tabu/src/tabu_search.cc

```diff
@@ -117,6 +117,7 @@ double TabuSearch::simpleTabuSearch(const vector<int> &starting,
                                     double startingObjective,
                                     long long ZCoeff,
                                     long long timeLimitInMilliSecs,
+                                    bool useTimeLimit,
```

Comment:
Why introduce this additional argument? (redundant with `timeLimitInMilliSecs`)

Reply:
To summarize our conversation: Checking for positive/negative timeLimitInMilliSecs does not always tell us if we should/should not care about timeout, so this additional argument is needed for simpleTabuSearch().